cordova-lib/src/cordova/util.js

```diff
@@ -297,7 +297,7 @@ function ensurePlatformOptionsCompatible (platformOptions) {
     }).map(function (arg) {
         return opts[arg] === true ?
             '--' + arg :
-            '--' + arg + '="' + opts[arg].toString() + '"';
+            '--' + arg + '=' + opts[arg].toString();
```

Comment:
If you're talking about this usage
```javascript
cordova.raw.build({
    platforms: ["android"],
    options: ["--debug", "--gradleArg=--no-daemon"]
})
```
i think we shouldn't do that but still need to maintain this scenario for compatibility.